Automattic/jetpack
# Preview different Plan UI's with a radio toggle

This PR adds a new component `DevCard` to the mix, which is rendered in the footer only if Jetpack is running in a **development version**.  

Right now it only allows you to toggle between different Jetpack plans so that you're able to view the **UI** differences.  

_**Toggling between these is not actually changing your plan, it is only updating the state for viewing/testing purposes of the UI**_ -- aka: Your plan is expected to revert back to your actual plan when the /site endpoint is queried next, so that is NOT a bug.  

There is no doubt that this can be improved/expanded a great deal in the future (and it will), but for now **this PR is meant to be a quick win for designers/growth when they are testing differences between plans**. 

**To Test:** 
- Checkout and view the `/wp-admin/admin.php?page=jetpack#/plans` tab.  
- Toggle between the plans.  The view should reflect accordingly.  
- Navigate to the dashboard.  Your plan should be updated along with the /site endpoint response.  
- Change the Jetpack version in jetpack.php to `4.8`, make sure the component does not render.  

![switchplan](https://cloud.githubusercontent.com/assets/7129409/23148130/cfc86fca-f7b1-11e6-8dc1-ae510d566553.gif)
